managed/CounterStrikeSharp.API/Server.cs

```diff
@@ -28,19 +28,51 @@ namespace CounterStrikeSharp.API
 {
     public class Server
     {
-        public static float TickInterval => NativeAPI.GetTickInterval();
+        public static float TickInterval => 0.015625f;
```

Comment:
This is hardcoded on native side as well, just saving a native call here for no real reason.